src/Evolution/Systems/RelativisticEuler/Tags.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+class DataVector;
+
+namespace RelativisticEuler {
+namespace Tags {
```

Comment:
These tags are system-agnostic (same for all hydro systems), can we put them somewhere more general?

Reply:
Good call. I include these here because we had some for `GrMhd`. However, it makes more sense to put them in a place analogous to `gr::Tags`.